code/ui/PlayerInfo.cs

```diff
@@ -38,16 +37,72 @@ public override void Tick()
                 return;
             }
 
+            RoleLabel.Text = $"{player.Role.ToString()}";
+        }
+    }
+
+    public class IndicatorsPanel : Panel
+    {
+        public BarPanel HealthBar { set; get; }
+        public BarPanel AmmoBar { set; get; }
+
+        public bool invisibleAmmo { set; get; }
```

Comment:
ok, this might be due to me being new to C#, but why is `invisibleAmmo` lower case?